src/components/Header/index.js

```diff
@@ -158,10 +161,24 @@ const Header = () => {
             {isAuth() ? (
               <Dropdown.Menu>
                 <Dropdown.Item onClick={handleLogout}>Log out</Dropdown.Item>
+                <Dropdown.Divider />
+                <Dropdown.Item onClick={() => setTheme("light")}>
+                  Light Theme
+                </Dropdown.Item>
+                <Dropdown.Item onClick={() => setTheme("dark")}>
+                  Dark Theme
+                </Dropdown.Item>
```

Comment:
Do you thing removing theme based on context will be good here?
For example if I choose light theme, I can be given option of dark theme only?

Reply:
Yeah sure we can do that

Reply:
I am asking it as a question. What do you think about it?

Reply:
Yeah that would be nice but right now we have setup only light theme dark theme, but we even had a discussion that their may be a need to add more theme variables in that case I think so we can keep it like this only.

Reply:
Thank you for clarification. Will merge it then.